sample-lnd.conf

```diff
@@ -467,10 +467,32 @@
 ; propagation 
 ; max-commit-fee-rate-anchors=10
 
-; A threshold defining the maximum amount of dust a given channel can have
-; after which forwarding and sending dust HTLC's to and from the channel will
-; fail. This amount is expressed in satoshis.
-; dust-threshold=500000
+; DEPRECATED: This value will be deprecated please use the new setting 
+; "channel-max-fee-exposure". This value is equivalent to the new fee exposure
+; limit but was removed because the name was ambigious.
+; dust-threshold=
+
+; This value replaces the old 'dust-threshold' setting and defines the maximum
+; amount of satoshis that a channel pays in fees in case the commitment 
+; transaction is broadcasted. This is enforced in both directions either when
+; we are the channel intiator hence paying the fees but also applies to the 
+; channel fee if we are NOT the channel initiator. It is
+; important to note that every HTLC adds fees to the channel state. Non-dust 
+; HTLCs add just a new output onto the commitment transaction whereas dust 
+; HTLCs are completely attributed the commitment fee. So this limit can also 
+; influence adding new HTLCs onto the state. When the limit is reached we won't 
+; allow any new HTLCs onto the channel state (outgoing and incoming). So 
+; choosing a right limit here must be done with caution. Moreover this is a 
+; limit for all channels universially meaning there is no difference made due to
+; the channel size. So it is recommended to use the default value. However if
+; you have a very small channel average size you might want to reduce this 
+; value.
+; Setting this value too low might cause force closes because the lightning
+; protocol has no way to roll back a channel state when your peer proposes a
+; channel update which exceeds this limit there are only two options to resolve
+; the situation. You can either increase the limit and the channel should be
```

Comment:
I kept it short to not explain the whole lightning state protocol here but regarding your questions:

2. Yes the channel would become inactive because the channel partner which exceeds the fee here will fail the channel with an internal error, depending on the lightning implementations some might already force close the channel if they receive an internal state maschine error from us, LND does not and there is a chance to bring this channel up again, if the limit is increased and the node restarted.

Reply:
This is not something we can address here:

But there is a UX problem in discovering that the channel has become inactive and then inspecting the log to find out what might have happened. Log might get cycled if the user is too late in inspecting and may lose this critical piece of info.
I was wondering if we can create a TODO or an issue to persist the reason for channel becoming inactive in the DB, so that discoverability of the issue is easier

Reply:
I think you are making a very good point here, we should at least record a channel state failure when reporting the channel to the user, will create an issue.